### PR TITLE
Getting rid of the "thread" table - first step

### PR DESCRIPTION
**Pull request by annando**
With this PR we mostly got rid of the "thread" table. There is still a query left and code needs to be removed. But since this worked fine for some days on my system, I think it is ready.

This final clean up will be part of another PR.

### CODE REVIEW

File: src/Model/Item.php [hunk 1]
```diff
@@ -899,8 +899,9 @@ public static function insert($item, $notify = false, $dontcache = false)
 
 			// If its a post that originated here then tag the thread as "mention"
 			if ($item['origin'] && $item['uid']) {
+				DBA::update('post-thread-user', ['mention' => true], ['uri-id' => $item['parent-uri-id'], 'uid' => $item['uid']]);
 				DBA::update('thread', ['mention' => true], ['iid' => $parent_id]);
```
Comment (MrPetovan) [inline src/Model/Item.php:903]: Shouldn't this call be removed?

File: update.php [hunk 2]
```diff
@@ -735,3 +735,27 @@ function update_1397()
...
+	}
+
+	if (!DBA::e("UPDATE `post-thread-user` INNER JOIN `thread` ON `thread`.`uid` = `post-thread-user`.`uid` AND `thread`.`uri-id` = `post-thread-user`.`uri-id`
+		SET `post-thread-user`.`mention` = `thread`.`mention`")) {
+			return Update::FAILED;
+	}
+
+	return Update::SUCCESS;
+
+}
```
Comment (MrPetovan) [inline update.php]: Code standards: Please add an EOL EOF

File: src/Model/Item.php (hunk 1 above)
Comment (annando) [reply src/Model/Item.php:903]: This will be done in the next PR. Then all "thread" related code will be removed.

File: update.php (hunk 2 above)
Comment (annando) [reply update.php]: Done

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**